tests/autostep-cli.tests/CommandInvocationTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Parsing;
+using System.Threading;
+using Xunit;
+using FluentAssertions;
+using static AutoStep.CommandLine.CreateProjectCommand;
+using FluentAssertions.Primitives;
+
+namespace AutoStep.CommandLine.Tests
+{
+    public class CommandInvocationTests
+    {
+        private static readonly CreateProjectDelegate successProjectCreation = MakeCreateProjectDelegateFromStatusCode(0);
+
+        private static CreateProjectDelegate MakeCreateProjectDelegateFromStatusCode(int returnStatusCode) =>
+            new CreateProjectDelegate((runArgs, iLogger) => returnStatusCode);
+
+        private static readonly AutoStepFiles _defaultMockAutoStepFiles = GetMockAutoStepFiles(successProjectCreation, successProjectCreation);
```

Comment:
One more variable name to fix.

Reply:
My bad, clearly did not pay enough attention.

Reply:
No worries.